### PR TITLE
Edited w/questions: public website intake form

### PR DESCRIPTION
**Pull request by jenniferlee-dsva**
@davidconlon @Karak888 - edited with some comments. let's discuss at our Monday's. CC @MickinSahni

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/public-websites-intake.md [hunk 1]
```diff
@@ -10,57 +10,64 @@ assignees: karak888
 # WIP
 
 ### Instructions
-- All requests for work needing to be fulfilled by a Public Websites resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into, so it is highly recommended that requests are submitted as soon as you're aware you will need help. Different types of requests may require more time in advance.
+- All teams requesting support or work that needs to be fulfilled by a Public Website resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into.
```
Comment (MickinSahni) [inline .github/ISSUE_TEMPLATE/public-websites-intake.md]: Consider leading off with a list of the types of situations/requests a team would have of Public Websites.

File: .github/ISSUE_TEMPLATE/public-websites-intake.md [hunk 2]
```diff
@@ -10,57 +10,64 @@ assignees: karak888
...
+ 5. Provide a link to product brief or outline: 
+ 6. Provide a link to wireframes, prototype: 
+ 7. Provide a link to any user research synthesis: 
+
+
+### Desired/target implementation date
+Is there an ETA for a particular launch or completion date when request needs to be completed (for instance, for a user research session or eBenefits migration)?
+
+
+### Type of support request
```
Comment (MickinSahni) [inline .github/ISSUE_TEMPLATE/public-websites-intake.md]: Consider moving this right under the types of requests Pubic Websites support so it's immediately clear what folks are asking the team.

**Review by MickinSahni — COMMENTED** [review]
Hey Jen, I added a couple comments. It’s helpful to quickly share a) Public Websites’ scope and b) the request. In general, we can likely limit reduce the number of words we’re using. Lmk if there are any follow-ups!

**Review by jenniferlee-dsva — COMMENTED** [review]
@Karak888 @davidconlon (CC @MickinSahni ) updated with a few additional edits re Mickin's comments

**Pull request merged**